openassessment/xblock/test/test_staff_area.py

```diff
@@ -1038,6 +1060,40 @@ def _setup_xblock_and_create_submission(self, xblock, **kwargs):
             'files_names': kwargs.get('files_names', [])
         }, ['staff'])
 
+    def _setup_team_assignment_xblock_and_create_team_submission(self, xblock, **kwargs):
+        """
+        A shortcut method to setup Team Assignment ORA xblock and add a team submission to the block.
```

Comment:
It feels like this can be refactored to leverage more shared logic with `_setup_xblock_and_create_submission` e.g.

```
_setup_xblock_and_create_submission(self, xblock, teams_enabled=False, **kwargs):
...
```